include/vara/Feature/Feature.h

```diff
@@ -278,6 +257,21 @@ class NumericFeature : public Feature {
 private:
   ValuesVariantType Values;
 };
+
+//===----------------------------------------------------------------------===//
+//                          RootFeature Class
+//===----------------------------------------------------------------------===//
+
+class RootFeature : public Feature {
+public:
+  RootFeature(string Name = "root")
```

Comment:
```suggestion
  explicit RootFeature(string Name = "root")
```
Ctors with only one parameter should be explicit to prevent unwanted implicit conversions.